### PR TITLE
Add command /tag-color

### PR DESCRIPTION
**Pull request by bbassie**
Adds a command to change the color of a tag, a fun thing for what I thought supporters and up to use. This also makes longer tags more readable.
factorio_Kn87h8jwlQ

### CODE REVIEW

**Comment by bbassie** [comment]
Not sure if I did the data part right.

File: config/expcore/command_color_parse.lua [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+-- @config Commands-Color-Parse
+
+local Commands = require 'expcore.commands' --- @dep expcore.commands
+local Colours = require 'utils.color_presets' --- @dep utils.color_presets
+
+Commands.add_parse('color',function(input, player, reject)
+  if not input then return end
+  local color = Colours[input]
+  if not color then
+    player.print("invalid")
```
Comment (Cooldude2606) [inline config/expcore/command_color_parse.lua:11]: This print is not required as reject will print the error message to the player.

**Review by Cooldude2606 — CHANGES_REQUESTED** [review]
Everything looks good, the datastore methods are used correctly.
You mention that you make longer tags look nicer, what change have you made to do this?

**Review by Cooldude2606 — APPROVED** [review]
Every looks good, ill be happy to merge this.

**Pull request merged**